### PR TITLE
Add introductions and links to SQL chapters

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

File: presto-docs/src/main/sphinx/language.rst [hunk 1]
```diff
@@ -1,9 +1,27 @@
+.. _sql-language:
+
 ************
 SQL Language
 ************
 
+Presto is an ANSI SQL compliant query engine. This standard compliance allows
+Presto user to integrate their favorite data tools, including BI and ETL tools
```
Comment (electrum) [inline presto-docs/src/main/sphinx/language.rst]: "allows the Presto user"
or
"allows Presto users"

File: presto-docs/src/main/sphinx/functions.rst [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+
 ***********************
 Functions and Operators
 ***********************
 
+This chapter describes the SQL functions and operators supported by Presto. They
+allow you to implement complex functionality and behavior of the SQL executed by
+Presto operating on the underlying data sources.
+
+Ensure to also consult our reference to the :ref:`supported SQL data
```
Comment (electrum) [inline presto-docs/src/main/sphinx/functions.rst]:
```
Ensure to also consult our reference to the
:doc:`supported SQL data types</language>`
and other general characteristics and a
:doc:`full SQL statement and syntax reference</sql>`
are available.

File: presto-docs/src/main/sphinx/language.rst [hunk 3]
```diff
@@ -1,9 +1,27 @@
...
+Presto users to integrate their favorite data tools, including BI and ETL tools
+with any underlying data source.
+
+Presto validates and translates the received SQL statements into the necessary
+operations on the connected data source.
+
+This chapter provides a reference to the supported SQL data types and other
+general characteristics of the SQL support of Presto.
+
+A :ref:`full SQL statement and syntax reference<sql-statement-syntax>` is
```
Comment (electrum) [inline presto-docs/src/main/sphinx/language.rst]:
```
A :doc:`full SQL statement and syntax reference</sql>` is
available in a separate chapter.

Advanced users can take it to the next level by
:doc:`using numerous SQL functions and operators</functions>`.

File: presto-docs/src/main/sphinx/functions.rst [hunk 4]
```diff
@@ -1,9 +1,19 @@
+.. _functions:
```
Comment (electrum) [inline presto-docs/src/main/sphinx/functions.rst]: We can link to the documentation pages directly with custom anchor text, so please remove all of these labels.

File: presto-docs/src/main/sphinx/language.rst (hunk 1 above)
Comment (mosabua) [reply presto-docs/src/main/sphinx/language.rst]: fixed

File: presto-docs/src/main/sphinx/functions.rst (hunk 4 above)
Comment (mosabua) [reply presto-docs/src/main/sphinx/functions.rst]: done

**Comment by mosabua** [comment]
All done as we chatted yesterday @electrum .. please merge

**Comment by electrum** [comment]
Merged, thanks!

**Pull request merged**